liferay-echo/liferay-portal
# LPS-198500 Migrate portal-workflow-task-web - main.js to purely javaScript

Motivation
=======
The motivation behind this ticket is the migration of js files that use AUI to files that don't use it since is deprecated.
https://liferay.atlassian.net/browse/LPS-198735

Steps to Verify
========
1. Go to Applications Menu -> Applications -> Workflow -> Process Builder -> Configuration -> look for Web Content Article -> Edit it -> change the input to Single Approver -> Save it
2. Go to Product Menu -> Content & Data -> Web Content -> click on (+) -> put a title and some content and click on Submit 
for Workflow
3. Click on the user-avatar link to open the personal menu -> Notifications -> open the Web content review -> click on the dropdown button at the up and right corner  -> Assign to me -> enter some chars and click Done (this will keep you in the same view)
4. In the same view click over the same dropdown button again and check that the Approve and Reject actions work correctly
